ochap-backend/src/main/java/ch/heigvd/amt/backend/routes/CategoryManager.java

```diff
@@ -16,6 +17,7 @@
 import java.util.Set;
 
 @Controller
+@PreAuthorize("hasAuthority('admin')")
```

Comment:
Why not use the more simple `@RolesAllowed("admin")` annotation instead of computing custom logic?

Mind that `@RolesAllowed` is part of JSR-250 and should be enabled with jsr250Enabled = true. (instead of `prePostEnable`).

Reply:
(This remark applies to all similar changes.)